shift_detector/precalculations/lda_gensim_tokenizer.py

```diff
@@ -51,40 +48,28 @@ def __hash__(self):
     def process(self, store):
         df1_texts, df2_texts = store[ColumnType.text]
 
-        if isinstance(self.cols, str):
-            if self.cols in df1_texts.columns:
-                col_names = [self.cols]
-        else:
-            for col in self.cols:
-                if col not in df1_texts.columns:
-                    raise ValueError('Given column is not contained in given datasets')
-            col_names = self.cols
+        for col in self.cols:
+            if col not in store.column_names(ColumnType.text):
+                raise ValueError("Following given column is not contained in given datasets: {}".format(col))
+        col_names = self.cols
 
         processed1 = {}
         processed2 = {}
 
-        for col in col_names:
-            tokenized1 = []
-            tokenized2 = []
-
-            for entry in df1_texts[col]:
+        def make_usable_list(texts):
```

Comment:
Can you make a descriptive name for this function or otherwise comment the method?